addons/payment/controllers/portal.py

```diff
@@ -376,6 +376,21 @@ def payment_confirm(self, tx_id, access_token, **kwargs):
             # Display the portal homepage to the user
             return request.redirect('/my/home')
 
+    @http.route('/payment/archive_token', type='json', auth='user')
+    def archive_token(self, token_id):
+        """ Check that a user has write access on a token and archive the token if so.
+        :param int token_id: The token to archive, as a `payment.token` id
+        :return: None
```

Comment:
```suggestion
        """ Check that a user has write access on a token and archive the token if so.
        
        :param int token_id: The token to archive, as a `payment.token` id
        :return: None
```
You lost this line along the way. It's important as it separates the docstring title from its body, the same as for commit messages.

Reply:
Just updated it, should be fine right now :slightly_smiling_face: